src/components/ExampleComponents.tsx

```diff
@@ -65,6 +67,14 @@ import {
 } from 'suomifi-ui-components';
 import { addDisplayNames } from 'components/ExampleComponentUtil';
 
+export const InlineAlert = (props: any): JSX.Element => (
```

Comment:
We want to use correct types, not want to have `any`.

This same applies to **SingleSelect** and I suggest to fix this at the same time.